### ISSUE TITLE
Cannot read files inside app.asar using fs

### ISSUE THREAD

**Issue by z1988hf** [electron/electron#20868]
let str = fs.readdirSync('C:\\Users\\Administrator\\AppData\\Local\\Programs\\contoso\\resources\\app.asar');
    console.log(str);

Errors:
Error: ENOTDIR: not a directory, scandir 'C:\Users\Administrator\AppData\Local\P
rograms\contoso\resources\app.asar'

Expected Behaviour:
return all files in it.

System:
win7
node:v10.15.0
electron:6.0.9

This is claimed to work.
![image](https://user-images.githubusercontent.com/25164182/67918571-c330d600-fb94-11e9-8d83-55b63dcc9818.png)

**Comment by z1988hf** [comment]
I used double \\ in path. It is just shown as \ in github.

**Comment by Liklei** [comment]
app.asar , limit to write and write

**Comment by Liklei** [comment]
app.asar , limit to write and read

**Comment by z1988hf** [comment]
> app.asar , limit to write and read

Yeah, I am reading it as a directory. If you mean only read as a file, then the electronjs docs is wrong.
